src/cordial_sound/scripts/wav_file_publisher.py

```diff
@@ -18,35 +21,66 @@ def __init__(self):
             'cordial/sound/wav/header_length',
             24
         )
-        rospy.Subscriber(rospy.get_param('cordial_sound/play_wav_topic'), String, self.play_wav_file)
-        self._sound_publisher = rospy.Publisher(rospy.get_param('cordial_sound/play_stream_topic'), Sound, queue_size=1)
+        rospy.Subscriber(rospy.get_param(
+            'cordial_sound/play_wav_topic'), String, self.play_wav_file)
+        self._sound_publisher = rospy.Publisher(rospy.get_param(
+            'cordial_sound/play_stream_topic'), Sound, queue_size=10)
         self._pyaudio = pyaudio.PyAudio()
 
+        self.audio_chunks = []
+
+    def single_split_sec(self, from_sec, to_sec, audio):
+        t1 = from_sec * 1 * 1000
+        t2 = to_sec * 1 * 1000
+        split_audio = audio[t1:t2]
+        self.audio_chunks.append(split_audio)
+
+    def multiple_split_sec(self, filename, seconds_per_split):
```

Comment:
Nit. It would be nice if these returned the audio chunks rather than modifying a member variable (the same goes for `single_split_sec`). I'm a big fan of functional programming in general.